.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,4 @@
+name: 'Octoclairvoyant CodeQL config'
+
+paths-ignore:
+  - 'cypress/**'
```

Comment:
This tells the action runner to ignore any files in the `cypress/**` directory.
Not sure if we can tell it to only ignore the Cypress `tsconfig.json` file though. [^docs]

[^docs]: https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#specifying-directories-to-scan

Reply:
I guess just putting the actual file rule would do the trick?

```suggestion
paths-ignore:
  - 'cypress/tsconfig.json'
```